### ISSUE TITLE
Error: Cannot find module './priorityTypeButton' - Witcher 3 Extension

### ISSUE THREAD

**Issue by Zeratoxx** [Nexus-Mods/Vortex#10741]
**Describe the bug**
The Vortex extension "Game: The Witcher 3" gets the error message "Error: Cannot find module './priorityTypeButton' [...]".
Symptoms: The game 'The Witcher 3' is not available in Vortex.

**To Reproduce**
1. Open Vortex (version 1.4.16)
2. Click on 'EXTENSIONS'
3. Scroll down to 'Game: The Witcher 3'
4. See an orange 'Failed'
5. Double-click 'Game: The Witcher 3'
6. See error message on the right side

**Expected behavior**
A clean startup without error.

**Screenshots**
![grafik](https://user-images.githubusercontent.com/35294329/149166822-3810197e-c2cb-4d6a-8620-d10325db24da.png)

**Platform (please complete the following information):**
 - OS: Windows 10 Pro, Version 10.0.19044 Build 19044
 - Vortex Version 1.4.16

**Additional context**
Vortex has been updated right before the unexpected behavior, didn't update since approximately one year.

**Comment by Zeratoxx** [comment]
Stacktrace (I inserted the "%AppData%", I don't want to leak my user account name):
```
Require stack:
- %AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\iconbarActions.js
- %AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\index.js
- C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\vortex-run\lib\index.js
- C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\renderer.js
- C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\index.html
```

**Comment by IDCs** [comment]
Hi @Zeratoxx - can you please close Vortex and delete the `%AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\` folder and start up Vortex again - it should re-download everything and hopefully fix this.

**Comment by QuentinDelignon** [comment]
Hey, I installed vortex yesterday and have the same problem, i run on windows 11 with the same version as you. Downgrading worked for a few sessions then eventually fell back to the same problem. It seems to be some missing front-end components called in nodejs .
Congrats to the devs on the soft anyway it's great!

**Comment by dorperj** [comment]
> Hi @Zeratoxx - can you please close Vortex and delete the `%AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\` folder and start up Vortex again - it should re-download everything and hopefully fix this.

I have the same problem and deleting the folder doesn't resolve the matter.

**Comment by Zeratoxx** [comment]
> Hi @Zeratoxx - can you please close Vortex and delete the `%AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\` folder and start up Vortex again - it should re-download everything and hopefully fix this.

@IDCs Done, the issue still exists. I created the issue with extension Version 1.3.7. Today, the issue still remains on extension version 1.3.8

**Comment by dorperj** [comment]
For now, I've commented out line 10 and line 41 of the file 'iconbarActions.js' in the %AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\ folder. Which results in a succesful import of the plugin.

**Comment by Zeratoxx** [comment]
@dorperj well, that removes the PriorityTypeButton in the script, what is the original purpose of it? Is it important? According to the naming of it, it somehow with the load order priority

**Comment by Zeratoxx** [comment]
Also, 

> For now, I've commented out line 10 and line 41 of the file 'iconbarActions.js' in the %AppData%\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\ folder. Which results in a succesful import of the plugin.
Worked for me also.

**Comment by panzerlop** [comment]
Thanks removing those lines also worked - also on W11

**Comment by Molzen** [comment]
Does not work for me on version 1.4.16
Getting the same error, even when removing the lines:
Failed to load: Cannot find module './priorityTypeButton'
Require stack:
- C:\Users\nikol\AppData\Roaming\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\iconbarActions.js
- C:\Users\nikol\AppData\Roaming\Vortex\plugins\Vortex Extension Update - The Witcher 3 v1.3.8\index.js
- C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\vortex-run\lib\index.js
- C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\renderer.js
- C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\index.html

**Comment by Zeratoxx** [comment]
> Thanks removing those lines also worked - also on W11 , I removed the lines entirely as wasn't sure if c# or c++


It's Javascript, but those Javascript files originally got compiled from typescript

**Comment by IDCs** [comment]
Hi all - as some of you may know - 1.3.8 has been out since the 1st of December 2021 - it baffles me why it would break after a month later which is why we suspected game extension corruption at first.

The priority button allows users to switch between fixed/flexible priority sorting (i.e. free or ascending) theoretically it shouldn't cause any issues if you remove it. Rather than manually deleting code, I would suggest to install our open alpha collections build instead which does not have this problem https://www.nexusmods.com/news/14608

While I investigate why this happened in the first place.

**Comment by IDCs** [comment]
Ok - so long story short; the issue was caused by a bug in our extension update system which was pulling the extension's file list from an incorrect branch. This became a problem because of a merge we've done earlier this week.

I've committed a workaround which will disable the priority sorting type button (yes I commented it out haha) to our release update branch which will ensure that users can still manage their mods and start the game. We will release a proper fix on Monday (new Vortex version 1.4.17) which will restore the button and ensure this sort of issue never happens again.

**Comment by dorperj** [comment]
> Ok - so long story short; the issue was caused by a bug in our extension update system which was pulling the extension's file list from an incorrect branch. This became a problem because of a merge we've done earlier this week.
> 
> I've committed a workaround which will disable the priority sorting type button (yes I commented it out haha) to our release update branch which will ensure that users can still manage their mods and start the game. We will release a proper fix on Monday (new Vortex version 1.4.17) which will restore the button and ensure this sort of issue never happens again.
> 
> Anyone affected by this issue - go to your `%appdata%/vortex/plugins` folder and delete any Witcher 3 extension you may have installed and start/restart Vortex - the fixed version should be downloaded automatically.

😂
